pkg/controller/atlasdeployment/advanced_deployment.go

```diff
@@ -222,3 +245,32 @@ func GetAllDeploymentNames(client mongodbatlas.Client, projectID string) ([]stri
 	}
 	return deploymentNames, nil
 }
+
+func normalizeInstanceSize(ctx *workflow.Context, currentInstanceSize string, autoscaling *mdbv1.AdvancedAutoScalingSpec) (string, error) {
```

Comment:
@helderjs Will this function handle the situation when the previous instance size is equal the current one from a CR? In that case the operator shouldn't send it.

Reply:
> when the previous instance size is equal the current one
@igor-karpukhin Do you mean the when instance size didn't change? if yes, does it make a difference to send the same value or not? will atlas try to "scale" to the same size?

Reply:
@helderjs Yes, it will try to scale to the same size. The problem is, if someone modifies anything else in the CR, we will send the same value for the instance size again.